mir-ci/mir_ci/test_drag_and_drop.py

```diff
@@ -40,27 +53,43 @@ class TestDragAndDrop:
         ],
     )
     async def test_source_and_dest_match(self, modern_server, app) -> None:
-        modern_server = DisplayServer(modern_server, add_extensions=VirtualPointer.required_extensions)
-        program = modern_server.program(apps.App(app))
+        extensions = VirtualPointer.required_extensions + ScreencopyTracker.required_extensions
+        server_instance = DisplayServer(modern_server, add_extensions=extensions)
+        program = server_instance.program(apps.App(app))
+
+        robot_settings = dedent(
+            f"""\
+            Library    {MIR_CI_PATH}/robot_libraries/WaylandHid.py
+            Library    {MIR_CI_PATH}/robot_libraries/Screencopy.py
+            Resource   {MIR_CI_PATH}/robot_resources/screencopy.resource
+        """
+        ).strip("\n")
 
-        robot_test_case = f"""Source and Destination Match
-            Sleep     {STARTUP_TIME}
-            Move Pointer To Absolute    40    40
-            Sleep     {A_SHORT_TIME}
-            Press LEFT Button
-            Sleep     {A_SHORT_TIME}
-            Move Pointer To Absolute    120    70
-            Sleep     {A_SHORT_TIME}
-            Move Pointer To Absolute    200    100
-            Sleep     {A_SHORT_TIME}
-            Release LEFT Button
+        robot_test_case = dedent(
+            f"""\
+            Source and Destination Match
+                Sleep     {STARTUP_TIME}
+                ${{regions}} =    Test Match    {MIR_CI_PATH}/robot_templates/drag_and_drop_src.png
```

Comment:
Shouldn't we wait instead of sleeping?

```suggestion
                ${{regions}} =    Wait Match    {MIR_CI_PATH}/robot_templates/drag_and_drop_src.png
```

Reply:
I don't think we need to wait here. We are certain that the relevant UI elements are displayed on the screen from the moment the application starts. The`Wait Match` would be useful in a more dynamic application where various UI layouts are displayed within a timeframe, and we are interested in a particular one.

Reply:
That's not really true, the application may delay startup for whatever reason. I'm actually working on such a case now (GTK insists on waiting on desktop portals). So the first step in a test should be establishing that the app we want to test initialized properly.

Remember the app starts asynchronously to Robot, the only thing we know is running by then is the server, but it may well be displaying black when Robot kicks in.

Reply:
Granted, we wait for the first frame to be ready, but we cannot make any assumptions about its contents. `Wait Match` is indeed a safer option. Actually, `Wait Match` subsumes `Test Match`, which raises the question of the usefulness of `Test Match`. Maybe we should always wait?

Reply:
Indeed, a match is just an immediate wait. I'd be fine just going with `Match...`.

Reply:
Done.